src/AppEntry.js

```diff
@@ -17,14 +18,18 @@ const InventoryApp = ({ logger }) => {
     };
   }, [logger]);
 
+  const queryClient = useMemo(() => new QueryClient(), []);
```

Comment:
While the `useMemo` will make the queryClient more stable, React might still throw it away sometimes. 

You can just have `const queryClient = new QueryClient();` outside the component within the `AppEntry` file. That should keep the query client the same as long as the AppEntry is mounted.

Reply:
neat! :)